### PR TITLE
slight modifications to soda/booze dispenser emag reagents

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
1) soda dispenser lost its booze, however - 
2) booze dispenser gained the 3 alcohols on emag that soda dispenser had, just for continuity
also moved iron over to the soda dispenser from the booze dispenser to thicken its emag results up

## Why It's Good For The Game
consistency between booze/soda dispenser reagents on emag; traitor gimmicks involving aphrodisiacs as a bartender are probably banworthy offenses since the first thing that comes to mind is using the aphrodisiacs and unwanted erp (which aphro chems count on) is bannable

## Changelog
:cl:
del: removed aphrodisiacs from booze dispenser emag results
tweak: moved some booze from the soda dispenser to the booze dispenser on emag, moved iron to the soda dispenser
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
might also add upgrade reagents to the dispenser as well but i don't have any good ideas and am open to suggestions

**Comment by Trilbyspaceclone** [comment]
The erp chems were their do to them making a few drinks they no longer do.

File: code/modules/reagents/chemistry/machinery/chem_dispenser.dm [hunk 1]
```diff
@@ -576,8 +573,9 @@
 		/datum/reagent/consumable/clownstears,
 		/datum/reagent/toxin/minttoxin,
 		/datum/reagent/consumable/ethanol/atomicbomb,
-		/datum/reagent/drug/aphrodisiac,
-		/datum/reagent/drug/aphrodisiacplus
+		/datum/reagent/consumable/ethanol/thirteenloko,
+		/datum/reagent/consumable/ethanol/changelingsting,
+		/datum/reagent/consumable/ethanol/whiskey_cola
```
Comment (Ghommie) [inline code/modules/reagents/chemistry/machinery/chem_dispenser.dm]: This one is literally just made of whiskey and cola and not that valuable. Could you please remove it?

Comment (ghost) [reply code/modules/reagents/chemistry/machinery/chem_dispenser.dm]: done

**Comment by ghost** [comment]
@Ghommie merge conflict fixed

File: code/modules/reagents/chemistry/machinery/chem_dispenser.dm [hunk 2]
```diff
@@ -511,12 +511,10 @@
 		/datum/reagent/consumable/peachjuice
 	)
 	emagged_reagents = list(
-		/datum/reagent/consumable/ethanol/thirteenloko,
-		/datum/reagent/consumable/ethanol/changelingsting,
-		/datum/reagent/consumable/ethanol/whiskey_cola,
 		/datum/reagent/toxin/mindbreaker,
 		/datum/reagent/toxin/staminatoxin,
 		/datum/reagent/medicine/cryoxadone
```
Comment (Ghommie) [inline code/modules/reagents/chemistry/machinery/chem_dispenser.dm]:
```suggestion
		/datum/reagent/medicine/cryoxadone,
```

File: code/modules/reagents/chemistry/machinery/chem_dispenser.dm [hunk 3]
```diff
@@ -571,11 +569,12 @@
...
 	upgrade_reagents3 = null
 	emagged_reagents = list(
-		/datum/reagent/iron,
 		/datum/reagent/consumable/ethanol/alexander,
 		/datum/reagent/consumable/clownstears,
 		/datum/reagent/toxin/minttoxin,
-		/datum/reagent/consumable/ethanol/atomicbomb
+		/datum/reagent/consumable/ethanol/atomicbomb,
+		/datum/reagent/consumable/ethanol/thirteenloko,
+		/datum/reagent/consumable/ethanol/changelingsting,
```
Comment (Ghommie) [inline code/modules/reagents/chemistry/machinery/chem_dispenser.dm]:
```suggestion
		/datum/reagent/consumable/ethanol/changelingsting
```

**Review by Ghommie — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
thanks for the cleanup i forgot about this pr

**Comment by ghost** [comment]
this is fine travis is just a baby

**Review by Ghommie — APPROVED** [review]

**Pull request merged**